lib/nostrum/shard/intents.ex

```diff
@@ -26,13 +31,18 @@ defmodule Nostrum.Shard.Intents do
 
   @spec get_enabled_intents :: integer()
   def get_enabled_intents do
-    enabled_intents = Application.get_env(:nostrum, :gateway_intents)
-
     # If no intents are passed in config, default to all being enabled.
-    if enabled_intents do
-      get_intent_value(enabled_intents)
-    else
-      get_intent_value(Keyword.keys(intent_values()))
+    enabled_intents = Application.get_env(:nostrum, :gateway_intents, :all)
```

Comment:
How do we feel about making the default be `:nonprivileged` so people who don't know about this setting still will have a working bot?

Reply:
I would prefer that as it's minimally invasive for existing bots' configs. I just kept it as all to preserve the original author's intention.

Reply:
👍 lets change it then

Reply:
oui